Crack_password.py

```diff
@@ -1,11 +1,39 @@
 from random import *
+
 user_pass = input("Enter your password: ")
-password = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j','k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't','u','v','w', 'x', 'y', 'z',]
+password = [
```

Comment:
For better performance a `set` or a `tuple` much quicker when reading password elements